File: Python/1065.py
# -*- coding: utf-8 -*-
lista = [ ]
somaPar = 0

lista.append(float(input()))
lista.append(float(input()))
lista.append(float(input()))
lista.append(float(input()))
lista.append(float(input()))

for p in lista:
    if p % 2 == 0:
        somaPar += 1

print(somaPar, "valores pares")

File: Java/1065.java
import java.io.IOException;
import java.util.Scanner;

public class Main {
 
    public static void main(String[] args) throws IOException {
        Scanner leitor = new Scanner(System.in);
        
        int vet[] = new int[5];
        int somaPar = 0;
        
        for (int i = 0; i < 5; i++) {
            vet[i] = leitor.nextInt();
        }
        
        for (int i = 0; i < 5; i++) {
            if (vet[i] % 2 == 0) {
                somaPar++;
            }
        }
        System.out.println(somaPar + " valores pares");
    }
 
}